app/backend/ipc/ipc_i18n_handler.js

```diff
@@ -29,19 +29,19 @@ class IpcI18nHandler {
     this.initIpcInterface();
   }
 
-  getLanguageFile(language) {
-      var fileName = path.join(__dirname, `../../../locales/${language}/translation.json`);
+  getLocaleFile(locale) {
+      var fileName = path.join(__dirname, `../../../locales/${locale}/translation.json`);
```

Comment:
Just to be consistent to use `locale` when we deal with app locale.

Reply:
Got it. Makes sense.